python/ray/autoscaler/azure/defaults.yaml

```diff
@@ -130,7 +130,7 @@ setup_commands:
     - echo 'eval "$(conda shell.bash hook)"' >> ~/.bashrc
```

Comment:
@gramhagen Is this still necessary with the Azure Data Science VM? Shouldn't Conda be preconfigured?

Reply:
yes for some reason the conda env isn't initialized when you ssh in (at least last I tried) and this command is needed to enable that.